src/content/api/events.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Events
+description: Events and related endpoints
+nav:
+  path: API/Events
+  order: 1
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Error from '../../components/Error.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+An event is an activity from an action which subsequently undergoes processing by job workers.
+
+## Webhook Notifications
+Only internal services will be calling this endpoint for Webhook notifications.
+The types can be `refund:completed`, `promotion:completed`, `promotion:updated` etc.
+
+## [Promotions](/api/promotions)
+When actions occur that could increment/complete a promotion, an event should be sent to Centrapay. Some events are native to Centrapay e.g `payment` . You do not need to send anything for native events, they will be handled by Centrapay.
+
+For your custom application events, you will need to define a `type` that the promotion will be incremented by.
+
+
+`type`s can be reused if the promotions should be incremented by the same action.
+
+
+When an action occurs that should increments a promotion, you need to send Centrapay an event at our [Create Event endpoint](/api/events) . We will respond with `200 OK` when we have received the event.
+
+Note that event processing is not live but should be processed within the next 2-3 minutes.
```

Comment:
I would rather put all this information in a guide. Can we remove this from here for now, and then can you create a story to write a guide later? We don't have all the information yet as we haven't designed the onboard to Loyalty Program API.

Reply:
Alternatively, you could write a guide now without that piece of information and then add it later.

Reply:
@Ray2126  Yep, I will create a sotry to write a guide later